.goreleaser.yml

```diff
@@ -110,35 +109,35 @@ docker_manifests:
   # Quay
   - name_template: "quay.io/authzed/spicedb:v{{ .Version }}"
     image_templates: [*amd_image_quay, *arm_image_quay]
-  - name_template: "quay.io/authzed/spicedb:latest"
+  - name_template: "quay.io/authzed/spicedb:{{ if .IsNightly }}devel{{ else }}latest{{ end }}"
```

Comment:
could we unintentionally push a development container as `latest` if this condition does not work?

Reply:
Yes; unfortunately I don't see a great way to test it (without publishing) since it will only build and tag the multi-arch manifest once it has pushed the other images. 

But it is verbatim from the goreleaser docs, see the bottom of this page: https://goreleaser.com/customization/nightlies/#what-is-skipped-when-using-nightly

Reply:
Yeah, I had the same question and concern, but if this is what they suggest, I guess its okay